File: leetcode/2024/python/58/lengthOfLastWord.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        j = len(s) - 1
        res = 0
        while j >= 0:
            if s[j] != " ":
                break
            j -= 1
        while j >= 0:
            if s[j] == " ":
                break
            res += 1
            j -= 1
        return res

File: leetcode/2024/cpp/58/lengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = s.size() - 1;
        while (s[i] == ' ')
        {
            i -= 1;
        }
        int res = 0;
        while (i >= 0  && s[i] != ' ')
        {
            res += 1;
            i -= 1;
        }
        return res;
    }
};
